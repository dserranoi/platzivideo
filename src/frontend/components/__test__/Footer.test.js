import React from 'react';
import { create } from 'react-test-renderer';
import { render, shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Footer from '../Footer';

configure({ adapter: new Adapter() })


describe('Footer Testing', () => {
    test('Match Snapshot', () => {
        const footer = create(<Footer />);
        expect(footer.toJSON()).toMatchSnapshot();
    });

    test('footer has class .footer', () => {
        const footer = shallow(<Footer />);
        const footerElem = footer.find('footer');
        expect(footerElem.hasClass('footer')).toBe(true);
    });
    
    test('footer has  anchor tags', () => {
        const footer = shallow(<Footer />);
        expect(footer.find('a')).toHaveLength(3);
    });
});