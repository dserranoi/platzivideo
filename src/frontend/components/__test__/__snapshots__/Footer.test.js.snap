// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Testing Match Snapshot 1`] = `
<footer
  className="footer"
>
  <a
    href="/"
  >
    Terminos de uso
  </a>
  <a
    href="/"
  >
    Declaraci√≥n de privacidad
  </a>
  <a
    href="/"
  >
    Centro de ayuda
  </a>
</footer>
`;

exports[`Footer Testing footer has class .footer 1`] = `undefined`;
